doc/classes/LabelSettings.xml

```diff
@@ -8,6 +8,127 @@
+		<method name="add_stacked_shadow">
+			<return type="void" />
+			<param index="0" name="index" type="int" default="-1" />
+			<description>
+				Adds a new stacked shadow to the label at the given [param index]. If [param index] is -1, the new stacked shadow will be added at the end of the list.
+			</description>
+		</method>
+		<method name="get_stacked_outline_color" qualifiers="const">
+			<return type="Color" />
+			<param index="0" name="index" type="int" />
+			<description>
+				Returns the color of the stacked outline at [param index].
+			</description>
+		</method>
+		<method name="get_stacked_outline_size" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="index" type="int" />
+			<description>
+				Returns the size of the stacked outline at [param index].
+			</description>
+		</method>
+		<method name="get_stacked_shadow_color" qualifiers="const">
+			<return type="Color" />
+			<param index="0" name="index" type="int" />
+			<description>
+				Returns the color of the stacked shadow at [param index].
+			</description>
+		</method>
+		<method name="get_stacked_shadow_offset" qualifiers="const">
+			<return type="Vector2" />
+			<param index="0" name="index" type="int" />
+			<description>
+				Returns the offset of the stacked shadow at [param index].
+			</description>
+		</method>
+		<method name="get_stacked_shadow_outline_size" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="index" type="int" />
+			<description>
+				Returns the outline size of the stacked shadow at [param index].
+			</description>
+		</method>
+		<method name="move_stacked_outline">
+			<return type="void" />
+			<param index="0" name="from_index" type="int" />
+			<param index="1" name="to_position" type="int" />
+			<description>
+				Moves the stacked outline at index [param from_index] to the given position [param to_position] in the array.
+			</description>
+		</method>
+		<method name="move_stacked_shadow">
+			<return type="void" />
+			<param index="0" name="from_index" type="int" />
+			<param index="1" name="to_position" type="int" />
+			<description>
+				Moves the stacked shadow at index [param from_index] to the given position [param to_position] in the array.
+			</description>
+		</method>
+		<method name="remove_stacked_outline">
+			<return type="void" />
+			<param index="0" name="index" type="int" />
+			<description>
+				Removes the stacked outline at index [param index].
+			</description>
+		</method>
+		<method name="remove_stacked_shadow">
+			<return type="void" />
+			<param index="0" name="index" type="int" />
+			<description>
+				Removes the stacked shadow at index [param index].
+			</description>
+		</method>
+		<method name="set_stacked_outline_color">
+			<return type="void" />
+			<param index="0" name="index" type="int" />
+			<param index="1" name="color" type="Color" />
+			<description>
+				Sets the color of the stacked outline identified by the given index.
```

Comment:
The existing description is fine with context, but I personally feel like it's incomplete because it omits the `color` parameter.
```suggestion
				Sets the color of the stacked outline identified by the given [param index] to [param color].
```
Maybe it's okay, though?
A similar suggestion would also apply also to the remaining methods below.